api

```diff
@@ -755,13 +755,11 @@ list_apps() { # $1 can be: installed, uninstalled, corrupted, cpu_installable, h
 }
 
 list_intersect() { #Outputs only the apps that appear in both stdin and in $1
-  #                      change \n to \|     |   remove last "\|"
-  grep -x "$(echo "$1" | sed -z 's/\n/\\|/g' | sed -z 's/\\|$/\n/g')"
+  comm -12 - <(echo "$1" | sort)
```

Comment:
This is fantastic! @theofficialgman how did you come across the `comm` command?

Reply:
its one of the coreutils packages. coreutils are the gnu untils installed on every system

documentation for all coreutils packages are here: https://www.gnu.org/software/coreutils/manual/

you can also just use the filelist and look at the binaries included in the deb https://packages.debian.org/bullseye/amd64/coreutils/filelist

Reply:
> its one of the coreutils packages. coreutils are the gnu untils installed on every system

We should probably still add it as a dependency of Pi-Apps..